src/org/centos/contra/Infra/Utils.groovy

```diff
@@ -373,6 +373,38 @@ def executeInAnsible(String playbook_path, String paramString, Boolean verbose,
     }
 }
 
+/**
+ * Execute shell scripts in the ansible container
+ * @param script_path
+ * @param paramString
+ * @param verbose - whether to include -vvv to the execution of the commands/playbooks in the shell scripts
+ * @param ansibleContainerName - name of ansible container in executing pod. default is 'ansible-executor'.
+ * @return
+ */
+def executeInShell(String script_path, String paramString, Boolean verbose, String ansibleContainerName){
+
+    String containerName = ansibleContainerName ?: 'ansible-executor'
```

Comment:
@robnester-rh @flowerncsu should this be changed as well to "ansibleExecutorContainerName" and "linchpinExecutorContainerName"? Likewise here  -  https://github.com/openshift/contra-hdsl/blob/master/src/org/centos/contra/Infra/Utils.groovy#L354
Just wanted to check if the new changes are just for scripts under vars/ as I am not very familiar with this change in https://github.com/openshift/contra-hdsl/pull/63 could you take a look? thanks.